packages/subiquity_client/test/types_test.dart

```diff
@@ -115,6 +115,25 @@ void main() {
     expect(ApplicationStatus.fromJson(json), status);
   });
 
+  test('os prober', () {
+    const os = OsProber(
+      long: 'Windows Boot Manager',
+      label: 'Windows',
+      type: 'efi',
+      subpath: '/efi/Microsoft/Boot/bootmgfw.efi',
+      version: '11',
+    );
+    final json = <String, dynamic>{
+      'long': 'Windows Boot Manager',
+      'label': 'Windows',
+      'type': 'efi',
+      'subpath': '/efi/Microsoft/Boot/bootmgfw.efi',
+      'version': '11',
+    };
+    expect(os.toJson(), equals(json));
+    expect(OsProber.fromJson(json), os);
+  });
+
   test('partition', () {
     const keyboard = Partition(
```

Comment:
Not strictly related to this change, but the name of the test object is misleading, it should be `partition`, not `keyboard`. Probably a copy-paste leftover from another test…

Reply:
i hope it's fine as a separate commit

Reply:
Yes, that's actually better, thanks!